### DISCUSSION TITLE
resample subject

### DISCUSSION THREAD

**Discussion by Chop1** [TorchIO-project/torchio#556] [Q&A] [answered]
Hello, I wanted to know if it is possible to resample a `subject`, that contains an image and a corresponding label image, while applying different interpolation.

I would like to use bspline for the actual image, but it could "create" new labels on the labels images, by trying to interpolates value. So, I would like to use the NearestNeighbor for the label image.

**Comment by fepegar** [comment, marked as answer]
Hi, @Chop1. Nearest neighbor interpolation is always used to resample instances of `LabelMap`, so you don't need to worry about this. I will try to make this more clear in the documentation.
